.circleci/test-deploy.yml

```diff
@@ -123,6 +123,12 @@ jobs:
                 command: |
                   echo "Test"
       - android/kill-emulators
+      - android/create-avd:
+          avd-name: test3
+          system-image: system-images;android-27;default;x86
```

Comment:
what's the rationale for using the oldest system image? should this be newer?

Reply:
I used the older system image to ensure that it installed this image while testing it

Reply:
i think this is fine, but perhaps in the future, we change this so it's testing the newest versions to catch any changes that would affect this